alembic/versions/c1d2e3f4a5b6_merge_v12_heads.py

```
@@ -0,0 +1,25 @@
+"""Merge v1.2 Alembic heads.
+
+Revision ID: c1d2e3f4a5b6
+Revises: 549c0b6c69a6, 6f9f9a8b7c6d
+Create Date: 2026-03-13
+"""
+
+from typing import Sequence, Union
```

Ruff (0.15.5)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)